src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -1863,9 +1863,36 @@ export class SavedObjectsRepository {
    */
   async openPointInTimeForType(
     type: string | string[],
-    { keepAlive = '5m', preference }: SavedObjectsOpenPointInTimeOptions = {}
+    {
+      keepAlive = '5m',
+      preference,
+      namespaces,
+      typeToNamespacesMap,
+    }: SavedObjectsOpenPointInTimeOptions = {}
   ): Promise<SavedObjectsOpenPointInTimeResponse> {
-    const types = Array.isArray(type) ? type : [type];
+    if (!type && !typeToNamespacesMap) {
+      throw SavedObjectsErrorHelpers.createBadRequestError(
+        'options.type must be a string or an array of strings'
+      );
+    } else if (namespaces?.length === 0 && !typeToNamespacesMap) {
+      throw SavedObjectsErrorHelpers.createBadRequestError(
```

Comment:
Those are the same checks than the ones performed in `find`

Reply:
See my comment in https://github.com/elastic/kibana/pull/109062#pullrequestreview-734416033, I think we should leave the `openPointInTimeForType ` function as-is in the SOR.